tests/helper/index.ts

```diff
@@ -0,0 +1,147 @@
+import ECPairFactory from "ecpair";
+import * as ecc from "@bitcoin-js/tiny-secp256k1-asmjs";
+import * as bitcoin from 'bitcoinjs-lib';
+import { StakingScripts } from "../../src/types/StakingScripts";
+import { StakingScriptData } from "../../src";
+
+const ECPair = ECPairFactory(ecc);
+
+export class DataGenerator {
+  private netWork: bitcoin.networks.Network;
```

Comment:
```suggestion
  private network: bitcoin.networks.Network;
```
weird capitalisation